idd/Energy+.idd.in

```diff
@@ -7604,6 +7604,54 @@ MaterialProperty:VariableThermalConductivity,
        \units W/m-K
        \type real
 
+MaterialProperty:VariableThermalAbsorptance,
+  A1 , \field Name
+       \required-field
+       \type alpha
+       \note The name of the dynamic coating material with variable thermal absorptance.
+  A2 , \field Reference Material Name
+       \required-field
+       \type object-list
+       \object-list MaterialName
+       \note Regular Material Name to which the additional properties will be added.
+       \note this the material name for the basic material properties.
+  A3 , \field Control Signal
+       \required-field
+       \type choice
+       \key SurfaceTemperature
+       \key SurfaceReceivedSolarRadiation
+       \key SpaceHeatingCoolingMode
+       \key Scheduled
+       \default SurfaceTemperature
+  A4 , \field Trigger Thermal Absorptance Function Name
+       \note A Curve:* or Table:Lookup object encoding the relationship between the trigger and surface thermal absorptance.
+  A5 ; \field Thermal Absorptance Schedule Name
+       \note only used when Control Signal = "Scheduled"
+
+MaterialProperty:VariableSolarAbsorptance,
+  A1 , \field Name
+       \required-field
+       \type alpha
+       \note The name of the dynamic coating material with variable solar absorptance.
+  A2 , \field Reference Material Name
+       \required-field
+       \type object-list
+       \object-list MaterialName
+       \note Regular Material Name to which the additional properties will be added.
+       \note this the material name for the basic material properties.
+  A3 , \field Control Signal
+       \required-field
+       \type choice
+       \key SurfaceTemperature
+       \key SurfaceReceivedSolarRadiation
+       \key SpaceHeatingCoolingMode
+       \key Scheduled
+       \default SurfaceTemperature
+  A4 , \field Trigger Solar Absorptance Function Name
+       \note A Curve:* or Table:Lookup object encoding the relationship between the trigger and surface solar absorptance.
+  A5 ; \field Solar Absorptance Schedule Name
+       \note only used when Control Signal = "Scheduled"
+
```

Comment:
Based on our discussion yesterday, I think it makes sense to combine these into a single `MaterialProperties:VariableAbsorptance` object with a single control signal and separate functions and schedules. If the function name is empty, it would mean that the absorptance component is not modulated.

Reply:
make sense. Will combine them into one field with separate function and schedule for thermal and solar